index.html

```diff
@@ -65,54 +77,70 @@ <h4>Skills</h4>
+                
+                </div>
+            </div>
             </div>
             <div class="works" id="work-2">
-                <img src="Works/work2.png" alt="Second work image">
-                <h4>Microverse Project 1</h4>
-                <p>This is a project with CSS. It was an introduction to the 
-                    use of linters and the process required for a code review.
-                </p>
-                <ul class="project-languages">
-                    <li class="project-languages-1">html</li>
-                    <li class="project-languages-1">css</li>
-                </ul>
-                <div class="n1"><button>See Project</button></div>
+                <div class="desktop-layout" id="second-works">
+                    <img src="Works/work2.png" alt="Second work image">
+                    <div class="desktop-layout-1">
+                        <h4>Microverse Project 1</h4>
+                        <p>This is a project with CSS. It was an introduction to the 
+                            use of linters and the process required for a code review.
+                        </p>
+                        <ul class="project-languages">
+                            <li class="project-languages-1">html</li>
+                            <li class="project-languages-1">css</li>
+                        </ul>
+                        <div class="n1"><button>See Project</button></div>
+                    </div>
+                </div>
             </div>
             <div class="works" id="work-3">
-                <img src="Works/work3.png" alt="Third work image">
-                <h4>Personal portfolio project</h4>
-                <p>This project was designed by Microverse to aid new
-                    students design their peronal portfolio. The basic languages
-                    for this project were HTML and CSS.
-                </p>
-                <ul class="project-languages">
-                    <li class="project-languages-1">html</li>
-                    <li class="project-languages-1">css</li>
-                </ul>
-                <div class="n1"><button>See Project</button></div>
+                <div class="desktop-layout">
+                    <img src="Works/work3.png" alt="Third work image">
+                    <div class="desktop-layout-1">
+                        <h4>Personal portfolio project</h4>
+                        <p>This project was designed by Microverse to aid new
+                            students design their peronal portfolio. The basic languages
+                            for this project were HTML and CSS.
+                        </p>
+                        <ul class="project-languages">
+                            <li class="project-languages-1">html</li>
+                            <li class="project-languages-1">css</li>
+                        </ul>
+                        <div class="n1"><button>See Project</button></div>
```

Comment:
- [ ] [_Optional_] You should add the 4th Project card even if you don't have a 4th Project right now, as is visible in the Figma template. You might not have a Project right now but soon you'll have a rapport full of Projects courtesy of Microverse. It would be easier to update than to add everything from scratch then :)